.gitlab-ci.yml

```diff
@@ -153,27 +145,25 @@ build-rust-doc-release:            &build
   <<:                              *build-only
   tags:
     - linux-docker
+  before_script:
+    - sccache -s
+    - ./scripts/build.sh
```

Comment:
is it really needed to build the wasm binaries here?

Reply:
yes